cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -961,7 +961,7 @@
     <name>metadata.updates.kafka.broker.list</name>
     <value>127.0.0.1:${kafka.bind.port}</value>
     <description>
-      CDAP Kafka service broker list to which metadata update notifications are published
+      Kafka service broker list to which metadata update notifications are published
```

Comment:
So, the CDAP was removed because (according to @dereklwood ) it can be either CDAP Kafka or Apache Kafka...can we leave it generic?

Why remove "service"?